tests/CommunityToolkit.Mvvm.UnitTests/Test_ICommandAttribute.cs

```diff
@@ -15,7 +15,7 @@ public partial class Test_ICommandAttribute
     [TestMethod]
     public async Task Test_ICommandAttribute_RelayCommand()
     {
-        MyViewModel? model = new();
+        MyViewModel model = new();
```

Comment:
Yeah that's expected, the fact the type is nullable or not doesn't actually change anything at runtime (because this is a reference type), the annotations are purely a build-time thing to inform developers, but don't have any functional impact. I just removed the annotation here since it was effectively incorrect: the model is immediately assigned and will never be `null`, so there was no reason why it should've been declared as nullable in the first place. That was likely just a typo I did when I originally wrote this unit test method a while back 🙂